tests/ctests/CMakeLists.txt

```diff
@@ -8,12 +8,10 @@ add_executable(runAleTests ${test_source})
 target_link_libraries(runAleTests ale ${GSL_LIBRARIES} ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} pthread)
 
 target_include_directories(runAleTests
-                           PRIVATE
-                           ${GSL_INCLUDE_DIRS}
-                           PUBLIC
-                           ${GSL_INCLUDE_DIRS}
-                           ${EIGEN3_INCLUDE_DIR}
-                           ${ALE_INCLUDE_DIRS})
+                            PUBLIC
+                            ${ALE_INCLUDE_DIRS}
+                            ${GSL_INCLUDE_DIRS}
+                            ${EIGEN3_INCLUDE_DIR})
```

Comment:
These can be removed and you can just link this as targets the same way they are linked in the top level cmakelist file.